static/js/components/Dashboard.js

```diff
@@ -2,9 +2,14 @@
 import React from 'react';
 import CourseList from './CourseList';
 import UserImage from './UserImage';
+import _ from 'lodash';
 
 class Dashboard extends React.Component {
 
+  formatId(id) {
+    return id ? `MMM${_.padStart(id.toString(), 6, '0')}` : undefined;
```

Comment:
yeah, that makes sense.

I thought there might be some reason why we want the actually number in the frontend, but we can always change it later if we do.